io/net/virt-net/network_virtualization.py

```diff
@@ -420,6 +423,44 @@ def test_vnic_auto_failover(self):
         else:
             self.cancel("Provide more backing device, only 1 given")
 
+    def test_rmdev_viosfailover(self):
+        '''
+        using mrdev and mkdev command to check vios failover
+        '''
+        self.login(self.vios_ip, self.vios_user, self.vios_pwd)
+        cmd = "lsmap -all -vnic -cpid %s" % self.lpar_id
```

Comment:
can we get exact vnic server, since there might be multiple for same lpars

Reply:
getting two active and backing.

Reply:
@smruti77 get vNIC interface device name in linux,  now for the lpar map the interface name with  `Client device name:env8` to get all active and backing and take first one for active device

Reply:
by using lap_id also getting the same thing,why need to use this "Client device name:env8"
again.

Reply:
@smruti77 same lpar can have multiple vNIC active device so ... each vNIC will have separate Client device name